modules/Microsoft.Cache/redis/deploy.bicep

```diff
@@ -253,7 +253,7 @@ module redisCache_privateEndpoints '../../Microsoft.Network/privateEndpoints/dep
     enableDefaultTelemetry: enableReferencedModulesTelemetry
     location: reference(split(privateEndpoint.subnetResourceId, '/subnets/')[0], '2020-06-01', 'Full').location
     lock: contains(privateEndpoint, 'lock') ? privateEndpoint.lock : lock
-    privateDnsZoneGroup: contains(privateEndpoint, 'privateDnsZoneGroup') ? privateEndpoint.privateDnsZoneGroup : {}
+    privateDnsZoneGroup: contains(privateEndpoint, 'privateDnsZoneGroup') ? privateEndpoint.privateDnsZoneGroup : []
```

Comment:
@DanielLarsenNZ Thanks for raising this. What was the outcome of the testing after change (if you've tested the change)? 
I've checked the referenced [module for private endpoints](https://github.com/Azure/ResourceModules/blob/main/modules/Microsoft.Network/privateEndpoints/deploy.bicep) and the input parameter is an object:

```bicep
@description('Optional. The private DNS zone group configuration used to associate the private endpoint with one or multiple private DNS zones. A DNS zone group can support up to 5 DNS zones.')
param privateDnsZoneGroup object = {}
```
However, I did also observe some inconsistencies around how are providing guidance on the usage of the parameter `privateEndpoints` where for the parameter `privateDnsZoneGroup` and the [documentation - parameter usage section for private endpoints](https://github.com/Azure/ResourceModules/tree/main/modules/Microsoft.Cache/redis#parameter-usage-privateendpoints) is not consistent as well where `privateDnsZoneResourceIds` should be `privateDNSResourceIds` 

This is how `privateDnsZoneGroup` should be used as per the private endpoint module

```json
        "privateDnsZoneGroup": {
            "value": {
                "privateDNSResourceIds": [
                    "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/privateDnsZones/privatelink.vaultcore.azure.net"
                ]
            }
        },
```

So I think there is a wider changed involved here

Reply:
Opened an issue to resolve the documentation problem here https://github.com/Azure/ResourceModules/issues/1802.

Reply:
PR to fix the documentation: #1805

Reply:
@eriqua I do not see the issue observed in this ticket. I would recommend we close this. @DanielLarsenNZ We can reopen this ticket and PR if required.

Reply:
Thanks @ahmadabdalla go ahead and close